docs/sources/mimir/configure/configuration-parameters/index.md

```diff
@@ -3569,6 +3581,33 @@ The `limits` block configures default and per-tenant limits imposed by component
 # CLI flag: -querier.active-series-results-max-size-bytes
 [active_series_results_max_size_bytes: <int> | default = 419430400]
 
+# (experimental) Defines labels for cost attribution, applied to metrics like
+# cortex_distributor_attributed_received_samples_total. Set to an empty string
+# to disable. Example: 'team,service' will produce metrics such as
+# cortex_distributor_attributed_received_samples_total{team='frontend',
+# service='api'}.
+# CLI flag: -validation.cost-attribution-labels
+[cost_attribution_labels: <string> | default = ""]
+
+# (experimental) Maximum number of cost attribution labels allowed per user.
+# CLI flag: -validation.max-cost-attribution-labels-per-user
+[max_cost_attribution_labels_per_user: <int> | default = 2]
+
+# (experimental) Maximum cardinality of cost attribution labels allowed per
+# user.
+# CLI flag: -validation.max-cost-attribution-cardinality-per-user
+[max_cost_attribution_cardinality_per_user: <int> | default = 10000]
+
+# (experimental) Cooldown period for cost attribution labels. Specifies the
+# duration the cost attribution remains in overflow before attempting a reset.
+# If the cardinality remains above the limit after this period, the system will
+# stay in overflow mode and extend the cooldown. Setting this value to 0
```

Comment:
```suggestion
# If the cardinality remains above the limit after this period, the system
# stays in overflow mode and extends the cooldown. Setting this value to 0
```

Reply:
all doc updates addressed in eebd0288658ce4e2c9447d89f5efc8f96e0878ee